WcaOnRails/spec/factories/user_groups.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user_group do
+    factory :delegate_regions_user_group do
+      name { "Delegate Regions" }
+      group_type { :delegate_regions }
+      is_active { true }
+      is_hidden { false }
+    end
+
+    factory :africa_region do
+      name { "Africa" }
+      group_type { :delegate_regions }
+      is_active { true }
+      is_hidden { false }
+      parent_group_id { (FactoryBot.create :delegate_regions_user_group).id }
```

Comment:
This parent ID is handled differently than all the other parent IDs below. Is there a reason for this?

Reply:
This was supposed to be the correct one. I've made it same in remaining as well. Thanks for pointing. Reason was that I initially (or rather copilot) wrote everything in the way like below, but it failed test case. So I tried changing like in this line.

Also, I'm using only :africa_region for test cases. I just added other regions so that it can be used somewhere in future if needed.